### PR TITLE
Link Status and other bug fixes

### PR DESCRIPTION
**Pull request by danipurwadi**
Changes done in rqt:
- Added Link Status field in the waypoint window as well as the individual aircraft field
- Added several other auto-close window upon ctrl+c (previously you would need to close manually)
- Bug fix on the timeout error
- Modified rqt such that the window is only generated after the valid id window is selected
- Fixed layout issue in waypoint window

### CODE REVIEW

File: launch/ogc_gndtest.launch [hunk 1]
```diff
@@ -37,11 +37,11 @@
 	</node>
 	
 	<!-- sbd link node -->
-	<node pkg="sbd" type="sbd_gnd_link.py" name="sbd_gnd_link"
+	<!-- <node pkg="sbd" type="sbd_gnd_link.py" name="sbd_gnd_link"
```
Comment (disgruntled-patzer) [inline launch/ogc_gndtest.launch]: Uncomment this

File: ogc_ws/src/identifiers/scripts/identifiers.py [hunk 2]
```diff
@@ -371,7 +371,7 @@ def edit_device(self, device):
 		selected_device["imei"] = device["imei"] if device["imei"] != ""else selected_device["imei"]
 		selected_device["rb_serial"] = device["rb_serial"] if device["rb_serial"] != "" else selected_device["rb_serial"]
 		selected_device["telegram_id"] = device["telegram_id"] if device["telegram_id"] != "" else selected_device["telegram_id"]
-		selected_device["syncthing_id"] = device["syncthing_id"] if device["syncthing_id"] != "" else selected_device["syncthing_id"]
+		# selected_device["syncthing_id"] = device["syncthing_id"] if device["syncthing_id"] != "" else selected_device["syncthing_id"]
```
Comment (disgruntled-patzer) [inline ogc_ws/src/identifiers/scripts/identifiers.py]: @rumesh986 can you check on this? Latest updates from the syncthing issue is that this was modified to fix issues with the admin server

File: ogc_ws/src/yonah_rqt/src/yonah_rqt/main_window.py [hunk 3]
```diff
@@ -73,15 +74,17 @@ def __init__(self, context):
...
         self.shortcuts()
-        self.CommandWindow.change_valid_ids()
+
         # Subscriber lists
         rospy.Subscriber("ogc/from_despatcher/regular", RegularPayload, self.regular_payload)
         rospy.Subscriber("ogc/yonahtext", String, self.status_text)
         rospy.Subscriber("ogc/from_despatcher/ondemand", String, self.ondemand)
         rospy.Subscriber("ogc/files/conflict", String, self.syncthing)
         rospy.Subscriber("ogc/feedback_to_rqt", LinkMessage, self.feedback_message)
+        rospy.Subscriber("ogc/stuff", String, self.link_status)
```
Comment (disgruntled-patzer) [inline ogc_ws/src/yonah_rqt/src/yonah_rqt/main_window.py]: The topic where the link status is published to is `ogc/from_despatcher/ondemand`, not `ogc/stuff`

**Review by disgruntled-patzer — CHANGES_REQUESTED** [review]
See comments above

File: ogc_ws/src/identifiers/scripts/identifiers.py (hunk 2 above)
Comment (rumesh986) [reply ogc_ws/src/identifiers/scripts/identifiers.py]: fixed along with the merge conflict

**Review by rumesh986 — APPROVED** [review]

**Review by huachen24 — APPROVED** [review]

**Review by disgruntled-patzer — APPROVED** [review]

**Pull request merged**